database/README.md

```diff
@@ -0,0 +1,168 @@
+# Database instructions
+
+## Requirements
+
+* Kubernetes cluster
+  
+  Follow the instructions to deploy a Kubernetes cluster from [Local Deployment]([kubernetes](https://kubernetes.io/docs/tasks/tools/#kubectl)).
+* go
+  
+  [Download and install go](https://go.dev/doc/install)
+* psql
+  ```bash
+  sudo dnf install postgresql
+  psql --version
+  psql (PostgreSQL) 15.6
+  ```
+* DBeaver Community
+  
+  A SQL client is recommended to interact easily with the Database. 
+  [DBeaver Community](https://dbeaver.io/).
+
+## Manual deployment
+
+The PostgreSQL manual deployment consists in 2 parts: First, the deployment of the Database infrastructure and then, using a script, create a table in the Database and populate it.
```

Comment:
It should be "consists **of two** parts". You change tense in the middle of the instructions, I have made the first part active in the suggestion below. Also, database should not be capitalized here.

```suggestion
The PostgreSQL manual deployment consists of two parts: First, deploy the database infrastructure and then, using a script, create a table in the database and populate it.
```